docker-compose.yml

```diff
@@ -65,18 +76,24 @@ services:
       memlock:
         soft: -1
         hard: -1
-    network_mode: host
+    ports:
+      - "9200:9200"
```

Comment:
Please change to "9200" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.